src/API/Util/PaymentMethods/Methods/AnyDay.php

```diff
@@ -0,0 +1,14 @@
+<?php
+
+namespace OnPay\API\Util\PaymentMethods\Methods;
+
+use OnPay\API\Util\PaymentMethods\Enums\Acquirers;
+use OnPay\API\Util\PaymentMethods\Enums\CurrencyCodes;
+
+/**
+ * @internal Internal use only
+ */
+final class AnyDay extends PaymentMethodAbstract {
```

Comment:
Anyday is not CamelCase. ApplePay however is. I think you've mixed them up :-)

Reply:
woops 👍